.gitignore

```diff
@@ -156,3 +156,4 @@ vendor/
 
 # CircleCI
 .circleci/generated_config.yml
+.circleci/storage
```

Comment:
nit: Please reinsert the final trailing newline in this file.

Reply:
Done